README.md

```diff
@@ -20,30 +20,94 @@
 
 [![Build Status](https://travis-ci.org/apache/incubator-openwhisk-runtime-java.svg?branch=master)](https://travis-ci.org/apache/incubator-openwhisk-runtime-java)
 
+## Changelogs
+- [Java 8 CHANGELOG.md](java8/CHANGELOG.md)
 
-### Give it a try today
-To use as a docker action
+
+## Quick Java Action
+A Java action is a Java program with a method called `main` that has the exact signature as follows:
```

Comment:
Is `main` necessary or can it be called something else?

Reply:
This is copy paste from the main docs. 
main is the default, if you read below it talks about how to specify a different method name `--main com.example.MyMain#methodName` using `#`